examples/esp_idf/hello/pytest/conftest.py

```diff
@@ -0,0 +1,13 @@
+import os
+import pytest
+
+@pytest.fixture(scope='session')
+def anyio_backend():
+    return 'trio'
+
+@pytest.fixture(scope="session")
+def credentials_file(request):
```

Comment:
The pytest-hil plugin already as the fixture for credentials file built in, so we can remove it here.